.eslintrc.js

```diff
@@ -40,22 +40,25 @@ module.exports = {
     'react',
     'prettier',
     'react-hooks',
-    'emotion',
+    '@emotion',
     'jsdoc',
     'react-ssr',
   ],
   rules: {
     'workday-custom-rules/restricted-imports': 'error',
-    '@typescript-eslint/class-name-casing': 'error',
```

Comment:
this was changed in 3.0.0 https://github.com/typescript-eslint/typescript-eslint/releases/tag/v3.0.0